Documentation/branch-management.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Branch management
-weight: 1
+weight: 1250
+description: etcd Git and project management
```

Comment:
can we keep just "etcd branch management" ?

Reply:
I can make that change :+1: